src/chrome/content/rules/Senate.gov.xml

```diff
@@ -15,7 +15,10 @@
 
 		<!-- The rules subdomain is all 403 via TLS -->
 		<!-- As is the Senate Intelligence Committee -->
-		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
```

Comment:
The point is that clarity is the most important factor. Given the new
exclusion for Rep. Sessions, having one exclusion encompass multiple
subdomains and others have just one is ugly and irregular.
On Mar 8, 2016 7:04 AM, "Ingvar Stepanyan" <notifications@github.com> wrote:

> In src/chrome/content/rules/Senate.gov.xml
> <https://github.com/EFForg/https-everywhere/pull/4260#discussion_r55370612>
> :
>
> > @@ -15,7 +15,10 @@
> >
> >  		<!-- The rules subdomain is all 403 via TLS -->
> >  		<!-- As is the Senate Intelligence Committee -->
> > -		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
> > +		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
> > +		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
>
> The style guide talks about "long strings of subdomains" as those can be
> expressed via <target host /> instead. It doesn't apply to exclusion
> pattern, especially for two subdomains.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EFForg/https-everywhere/pull/4260/files#r55370612>.
>